typescript/listeners/listeners.py

```diff
@@ -227,6 +230,11 @@ def on_close(self, view):
             #         most_recent_used_file_list.remove(info)
             # notify the server that the file is closed
             cli.service.close(file_name)
+        if TypeScriptEventListener.about_to_close_all:
```

Comment:
Leave a comment immediately above explaining what you are trying to avoid/accomplish